osquery/tables/networking/darwin/tests/wifi_tests.mm

```diff
@@ -22,27 +23,48 @@
 class WifiNetworksTest : public testing::Test {};
 
 TEST_F(WifiNetworksTest, test_parse_wifi_networks) {
-  std::string path = kTestDataPath + "test_airport.plist";
+  auto qd = SQL::selectAllFrom("os_version");
+  if (qd.size() != 1) {
+    FAIL() << "Couldn't determine OS X version";
+  }
+
+  std::string key = nullptr;
+  std::string path = nullptr;
+  if (qd.front().at("major") == "10" && qd.front().at("minor") == "9") {
+    key = "RememberedNetworks";
+    path = kTestDataPath + "test_airport_darwin_10_9.plist";
+  } else {
+    key = "RememberedNetworks";
+    path = kTestDataPath + "test_airport_darwin_10_9.plist";
```

Comment:
d'oh! copy pasta error, it should be `test_airport.plist` which already exists for 10.10 and later. Do you prefer the longer name?